praw/objects.py

```diff
@@ -1899,6 +1902,34 @@ def remove_editor(self, username, *args, **kwargs):
         return self.add_editor(username=username, _delete=True, *args,
                                **kwargs)
 
+    def revisions(self, **kwargs):
+        """Return a dictionary of all page revisions.
+
+        Additional parameters are passed to
+        :meth:`~praw.__init__.BaseReddit.request_json` to be used as
+        URL parameters.
+        """
+        url = self.reddit_session.config['wiki_page_revisions'].format(
+                subreddit=six.text_type(self.subreddit),
+                page=self.page.lower())
+
+        prev_use_oauth = self.reddit_session._use_oauth
+        self.reddit_session._use_oauth = True
+        response = self.reddit_session.request_json(url, params=kwargs)
```

Comment:
`get_content` sound way cooler than the ugliness I've done. I'll try to use it, once I get passed a recursion problem that has come up.